Source/Scene/QuadtreePrimitive.js

```diff
@@ -1,11 +1,16 @@
 /*global define*/
 define([
+        '../Core/Cartesian3',
```

Comment:
@kring do you have any feedback on changes to this file or `QuadtreeTile.js`?

We're not thrilled with the one-off manual time slicing, but that will be addressed in #2655.